Source/WebCore/page/SecurityOriginData.cpp

```diff
@@ -39,15 +39,15 @@ namespace WebCore {
 
 String SecurityOriginData::toString() const
 {
-    if (protocol == "file"_s)
+    if (protocol() == "file"_s)
```

Comment:
Given that these getters are not necessarily super cheap, we may want to consider caching their result in variables to avoid calling them repeatedly in this function.